src/other/Table.js

```diff
@@ -322,9 +322,11 @@
             .remove()
         ;
 
-        if (this.data().length) {
-            this.fixedLabelsUpdate(domNode, element);
+        if (!this.data().length || !this.columns().length) {
+            this.fixedHeader(false);
+            this.fixedColumn(false);
```

Comment:
If I clear out the data and then load more in - I would expect it to still have fixed headers/columns.